docs/examples.md

```diff
@@ -233,7 +233,7 @@ jobs:
           fail-on-severity: critical
           deny-licenses: LGPL-2.0, BSD-2-Clause
           comment-summary-in-pr: always
-          allow-dependencies-licenses: 'pkg:npm/loadash, pkg:pypi/requests'
+          allow-dependencies-licenses: 'pkg:npm/lodash, pkg:pypi/requests'
```

Comment:
this was odd... 

afaict, `loadash` isn't a package real people are supposed to use:
https://www.npmjs.com/package/loadash